tests/common/test_cgroups.py

```diff
@@ -17,8 +17,8 @@
 
 from __future__ import print_function
 
-from azurelinuxagent.common.cgroups import CGroupsTelemetry, CGroups, CGroupsException, BASE_CGROUPS, Cpu, Memory, \
-    DEFAULT_MEM_LIMIT_MIN_MB
+from azurelinuxagent.common.cgroups import CGroupsTelemetry, CGroups, CGroupsException, CGroupsLimits, BASE_CGROUPS, \
+    Cpu, Memory, DEFAULT_MEM_LIMIT_MIN_MB
```

Comment:
Add unit tests for CGroupsLimits that exercise

- missing threshold map
- partial threshold map
- fully specified threshold map
- empty threshold map

Reply:
Done. Thanks for reminding me to do this.